sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptions extends AzureServiceClientOptions {
+    // (undocumented)
+    baseUri?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.RequestOptionsBase {
```

Comment:
preferred to extend `OperationOptions` instead

Reply:
It looks like this would be a modification in the swagger

---
In reply to: [447255424](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447255424) [](ancestors = 447255424)

Reply:
If a swagger change is necessary, can you let me know what change is needed so we can make that change for ADT GA?

---
In reply to: [454684353](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r454684353) [](ancestors = 454684353,447255424)